application.rb

```diff
@@ -0,0 +1,57 @@
+class Application
+  def initialize(scorecard, round)
+    @scorecard = []
+    @round = 1 
+  end
+
+  def first_throw
+    while @round <= 11 do
+    if @round == 11
+      strike_calculator
+      puts "Thanks for playing. Your total score is #{@scorecard.sum}"
+      break
+    end
+    
+    puts "Please enter the score from your first roll from @round #{@round}"  
+    @roll_1 = gets.chomp.to_i
+      if @roll_1 < 10
+        @scorecard << @roll_1
+        second_throw
+      else
+        @scorecard << @roll_1        
+        @round += 1 
+      end
+    end
+  end
+
```

Comment:
It would be useful to have some comments so that a reviewer can quickly identify what a method does.